docassemble/tclpgoogledocsmerger/data/questions/search_and_download.yml

```diff
@@ -80,6 +91,17 @@ fields:
     required: False
     code: |
       timeline_sub_types
+script: |
+  <script type="text/javascript">
+      $(document).ready(function() {
+          $('#aW5kdXN0cnlfdHlwZQ').multiselect({enableCaseInsensitiveFiltering: true, inheritClass: true});
+          $('#bnpfc2NvcGU').multiselect({enableCaseInsensitiveFiltering: true, inheritClass: true});
+          $('#Y29wXzI2').multiselect({enableCaseInsensitiveFiltering: true, inheritClass: true});
+          $('#aW5kdXN0cnlfZ3JvdXA').multiselect({enableCaseInsensitiveFiltering: true, inheritClass: true});
+          $('#dGltZWxpbmVfdHlwZQ').multiselect({enableCaseInsensitiveFiltering: true, inheritClass: true});       
+          $('#dGltZWxpbmVfc3ViX3R5cGU').multiselect({enableCaseInsensitiveFiltering: true, inheritClass: true});       
```

Comment:
Are these hard coded to the label / field names? Should keep that in mind as we update language around those.

Reply:
Unfortunately yes, for now. Would like to move this into the toolbox and perhaps make it apply with defaults to every multi select instead of needing to apply by id.